contracts/strategy/anchor/AnchorStrategy.sol

```diff
@@ -189,6 +197,11 @@ contract AnchorStrategy is IStrategy, AccessControl {
         );
     }
 
+    // @inheritdoc IStrategy
+    function disinvest() external virtual override(IStrategy) onlyManager {
```

Comment:
What's this for?

Reply:
this part is WIP from this morning, realized we were missing a InitRedeemDeposit call in the disinvest case of the new updateInvested function and i figured i'd make a new function to put the logic in there

like that we have an invest and disinvest function, not just an invest function

Reply:
nevermind, the withdrawToVault function does exactly this, removed this